server/apps/newsletter/beehiiv.py

```
@@ -0,0 +1,155 @@
+        Validate the format of an email address using Django's built-in validator.
+        
+        Args:
+            email (str): Email address to validate
+            
+        Raises:
+            ValueError: If the email is not in a valid format
+        """
+        try:
+            validate_email(email)
+        except ValidationError:
+            raise ValueError(f"Invalid email format: {email}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)